src/main/resources/wiki.json

```diff
@@ -29,5 +29,31 @@
 	"NETHER_GOLD_PAN": "Nether-Gold-Pan",
 	"PICKAXE_OF_CONTAINMENT": "Pickaxe-of-Containment",
 	"SEISMIC_AXE": "Seismic-Axe",
-	"SMELTERS_PICKAXE": "Smelter's-Pickaxe"
+	"SMELTERS_PICKAXE": "Smelter's-Pickaxe",
+	"IRON_DUST": "Iron-Dust",
+	"GOLD_DUST": "Gold-Dust",
+	"IRON_INGOT": "Iron-Ingot",
```

Comment:
This file will only work for Slimefun items, not vanilla ones.

Reply:
Oopsie, my bad